src/medium_data_bakeoff/ingredients/dask_auto_optimized.py

```diff
@@ -0,0 +1,16 @@
+import time
+
+import dask_expr as dd
+import ray
+from ray.util.dask import enable_dask_on_ray
+
+
+def bake(dataset: str) -> float:
+    ray.init()
+    with enable_dask_on_ray():
```

Comment:
I think you switched the names between this module and `dask_auto_optimized_on_ray.py`?

Reply:
I sure did.  Will swap them